conans/test/functional/toolchains/ios/test_using_cmake.py

```diff
@@ -55,6 +55,7 @@ def package(self):
             """)
         })
 
+    @pytest.mark.tool_cmake(version="3.20")
```

Comment:
We will need to add it in `conftest.py`, e.g.:

```python
        "3.20": {
            "path": {'Windows': 'C:/cmake/cmake-3.20.0-win64-x64/bin',
                     'Darwin': '/Users/jenkins/cmake/cmake-3.20.0/bin',
                     'Linux': '/usr/share/cmake-3.20.0/bin'}
        },
```

Anyway, let's wait for CI members to check out the final installation folders.